ci/azure-piplines.yml

```diff
@@ -0,0 +1,86 @@
+jobs:
+- job: hera_cal
+  pool:
+    vmImage: "ubuntu-latest"
+  variables:
+    PYTHON: "3.9"
+    ENV_NAME: hera_cal_tests
+    WITH_SUDO: true
+
+  steps:
+  - bash: |
+      cd ../
+      git clone https://github.com/HERA-Team/hera_cal.git
+      cd s
```

Comment:
Is the directory really named `s` on azure for the current project?

Reply:
yes! I assumed it was going to be pyuvdata, but that errored so I dug up the documentation and it's `s`. Totally crazy.